jest.config.js

```diff
@@ -0,0 +1,195 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/en/configuration.html
+ */
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
```

Comment:
Could you please remove all the commented out noise from the jest config template?

Reply:
Done.